model.py

```diff
@@ -384,15 +403,15 @@ def test(self, args):
         """Test pix2pix"""
         tf.initialize_all_variables().run()
 
-        sample_files = glob('./datasets/{}/val/*.jpg'.format(self.dataset_name))
+        sample_files = glob('./datasets/{}/val/*.jpg'.format(self.dataset_name))+glob('./datasets/{}/val/*.png'.format(self.dataset_name))
```

Comment:
I think this line is rather confusing, I should make it clear that this project only support `.jpg` files.